controllers/configuration_controller.go

```diff
@@ -98,6 +103,10 @@ const (
 
 const (
 	GitCredsKnownHosts = "known_hosts"
+	// Terraform credentials
+	TerraformCredentials = "credentials.tfrc.json"
+	// Terraform Registry
+	TerraformRegistry = "terraformrc"
```

Comment:
Why the var is named TerraformRegistry?

Reply:
the variable in named terraformregistry because the secret contains the details of the custom  terraform registry URL we want to connect to.

Reply:
But this rc file can contains other configurations too. Only `TerraformRegistry` is inaccurate.